docs/api/generate-apikey.md

```diff
@@ -0,0 +1,28 @@
+---
+id: generate-apikey
+title: Generate API Key
+sidebar_label: Generate API Key
+---
+
+## Introduction
+To perform activities with Conviso CLI, Conviso Platform Integrations and also Conviso API, it is important to generate an API Key to authenticate your user. 
+
+## Usage
+Follow these steps as shown in the gif:
+
+1. Log in to Conviso Platform and go to **"Edit Profile"** in the upper right corner;
+2. Click on API Key tab and **Generate** button to create your key;
+3. Click **Confirm** after generating the key for successful creation.
```

Comment:
i think would be interesting if we iterate for the user that every time he click on "generate api key" the "old" api key will be invalid